### PR TITLE
Paleodiversity

### PR DESCRIPTION
**Pull request by mcranium**
I have created a style for the German journal "Paleodiversity" using the visual editor from citationstyles.org (plus some changes in the code editor. 
I tried to validate the file at https://validator.citationstyles.org/ but I ran into three issues which I could not solve on my own (line 119, 626 and 770). Can someone check for these mistakes? I am not a professional programmer. 
I tested the csl file it in Zotero/Libreoffice and it ran without any problems.

### CODE REVIEW

File: paleodiversity.csl [hunk 1]
```diff
@@ -0,0 +1,771 @@
...
+          <names variable="editor translator" delimiter=", ">
+            <name and="symbol" initialize-with=". " delimiter=", "/>
+            <label form="short" prefix=", " text-case="title"/>
+          </names>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author" font-variant="normal">
```
Comment (POBrien333) [inline paleodiversity.csl]: delete font-variant="normal" here.

File: paleodiversity.csl [hunk 2]
```diff
@@ -0,0 +1,771 @@
...
+    </names>
+  </macro>
+  <macro name="author-short">
+    <choose>
+      <if type="patent" variable="number" match="all">
+        <text macro="patent-number"/>
+      </if>
+      <else>
+        <names variable="author" font-variant="normal">
+          <et-al font-variant="normal"/>
```
Comment (POBrien333) [inline paleodiversity.csl]: Delete the whole of line 118.
It's not doing anything and et-al could only be after the <name part. here it comes first.

File: paleodiversity.csl [hunk 3]
```diff
@@ -0,0 +1,771 @@
...
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <group prefix=":">
+      <choose>
+        <if locator="chapter">
+          <label text-case="capitalize-first" prefix=" " variable="locator"/>
+        </if>
+        <else/>
```
Comment (POBrien333) [inline paleodiversity.csl]: remove line 626 with `<else/>`. Not doing anything and it's giving you an error.

File: paleodiversity.csl [hunk 4]
```diff
@@ -0,0 +1,771 @@
...
+        <else-if variable="original-date">
+          <group prefix=" (" suffix=")" delimiter=" ">
+            <text macro="original-published"/>
+            <text macro="original-date"/>
+          </group>
+        </else-if>
+      </choose>
+    </layout>
+  </bibliography>
+  <citation et-al-min="3" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1"/>
```
Comment (POBrien333) [inline paleodiversity.csl]: delete line 770. You can't have to <citation sections in your style.

File: paleodiversity.csl (hunk 1 above)
Comment (rmzelle) [reply paleodiversity.csl]:
```suggestion
    <names variable="author">
```

File: paleodiversity.csl (hunk 2 above)
Comment (rmzelle) [reply paleodiversity.csl]:
```suggestion
```

File: paleodiversity.csl (hunk 3 above)
Comment (rmzelle) [reply paleodiversity.csl]:
```suggestion
```

File: paleodiversity.csl (hunk 4 above)
Comment (rmzelle) [reply paleodiversity.csl]:
```suggestion
```

**Comment by rmzelle** [comment]
Thanks!

**Comment by mcranium** [comment]
Thank you so much for your help guys! You are truly amazing!

**Comment by rmzelle** [comment]
You're welcome!

**Pull request merged**